lib/Test/Nginx/Socket.pm

```diff
@@ -2686,6 +2707,27 @@ will produce the following line (to C<stderr>) while running this test block:
 You need to remember to set the C<TEST_NGINX_NO_CLEAN> environment to 1 to prevent the nginx
 and other processes from quitting automatically upon test exits.
 
+=head2 response_body_filters
+
+Transform value for the body of the submitted request.
+
+    sub uppercase {
+        return uc $shift;
+    }
+    --- response_body_filters eval
+    \&main::uppercase
```

Comment:
@agentzh , i try `\&CORE::uc`, it also raise the error `Undefined subroutine &CORE::uc called`
It's strange. checkout this [perl document](http://perldoc.perl.org/CORE.html), i'm not sure whether can call reference to CORE::XX. But i test this POC as the following, it's okay.
```perl
$x = \&CORE::uc;
print $x->("abcd");
```

Do you have any idea for this?